src/Components/SavedSearchAlert/Components/ConfirmationStepFooter.tsx

```diff
@@ -0,0 +1,93 @@
+import { Button, Flex, SkeletonBox } from "@artsy/palette"
+import { SystemQueryRenderer } from "System/Relay/SystemQueryRenderer"
+import { FC } from "react"
+import { graphql } from "react-relay"
+import {
+  ConfirmationStepFooterQuery,
+  ConfirmationStepFooterQuery$data,
+} from "__generated__/ConfirmationStepFooterQuery.graphql"
+import { RouterLink } from "System/Router/RouterLink"
+import { useTranslation } from "react-i18next"
+
+interface ConfirmationStepFooterProps {
+  me: ConfirmationStepFooterQuery$data["me"]
+}
+
+const ConfirmationStepFooter: FC<ConfirmationStepFooterProps> = ({ me }) => {
+  const { t } = useTranslation()
+  const savedSearch = me?.savedSearch
+
+  return (
+    <Flex flexDirection={["column", "row"]}>
+      <Button
+        width="100%"
+        mr={[0, 1]}
+        mb={[1, 0]}
+        // @ts-ignore
```

Comment:
Is there a better way to do it without `ts-ignore`? I've took inspiration from here: https://github.com/artsy/force/blob/main/src/Apps/CollectorProfile/Routes/Saves2/Components/ArtworkListEmptyState.tsx#L36

Reply:
@nickskalkin I take it that the answer was that there isn't a better way right now? (I was [also wondering](https://artsy.slack.com/archives/CP9P4KR35/p1687547580653299?thread_ts=1687543308.378649&cid=CP9P4KR35) about `<Button as={…} />` recently)